src/core/RecyclerListView.tsx

```diff
@@ -60,7 +62,7 @@ const refreshRequestDebouncer = debounce((executable: () => void) => {
 });
 
 /***
- * This is the main component, please refer to samples to understand how to use.
+ * This is the maDebugHandlerin component, please refer to samples to understand how to use.
```

Comment:
Rename to original.. Seems global rename did this.

Reply:
yeah will fix this, sorry about this.